pallets/loans/src/functions.rs

```diff
@@ -460,7 +471,7 @@ impl<T: Config> Pallet<T> {
 				false => Ok(old_pv
 					.checked_sub(&new_pv)
 					.and_then(|negative_diff| nav.latest.checked_sub(&negative_diff))
-					.unwrap_or_else(Zero::zero)),
+					.unwrap_or_else(Zero::zero)), // Error instead?
```

Comment:
I mean, repaying more than the total nav value should not be possible, I guess

Reply:
Total nav value is separate from this. The NAV represents the valuation of the loan, this can be different from the outstanding debt. E.g. a loan can be written down reducing the valuation (which is basically saying, there's an expectation that this loan won't be fully repaid), but the debt will then reduce the same.

So it should not be possible to repay more than the outstanding debt, but the present value does not matter for this.

Reply:
Ok, understand... So in case it should not be possible, we can return an error instead there, and it could be fine